Train.py

```diff
@@ -62,6 +62,10 @@ def run_net(data_index):
 
                 # Logging Loss
                 epoch_train_loss.add(data.train_index.ctr, batch.loss.data[0])
+                epoch_train_loss.export_csv(
+                    'logs/epoch%02d_train_loss.csv' %
```

Comment:
In my experiments this doesn't really affect training speed (still getting 200-250 Hz). I think a better idea would be to move this in with the if ARGS.display stuff, that way it's updated less frequently (user can specify).